pxeboot

```diff
@@ -427,7 +427,7 @@ def try_pxy_boot():
     if not os.path.exists("/www/initrd.img"):
         shutil.copy(os.path.join(base, "initrd.img"), "/www/initrd.img")
     if not os.path.exists("/www/mnt") and os.path.exists("/var/ftp/mnt/images"):
-        common_bf.run("ln -s /www/mnt /var/ftp/mnt")
+        common_bf.run("ln -s /var/ftp/mnt /www/mnt")
```

Comment:
I'm surprised by this since we've been using the pxeboot script for quite some time and it was working. Did you test the change? Is it required? Do you have insight on why it was working in the past?

Reply:
> Did you test the change? Is it required?

Yes, I had to type the command manually to continue the instllation.
Here is the steps to reproduce.

I have the iso and efiboot.img in /iso on the host server.
[root@host105 ~]# ll /iso
total 18926664
-rw-r--r--. 1 root            root                7282688 Aug 25 01:49 efiboot.img
-rw-r--r--. 1 qemu            qemu            11465129984 Aug 23 19:26 rhel-8.6-x86_64-dvd.iso
-rw-r--r--. 1 matsuda-daisuke matsuda-daisuke  7908491264 Aug 25 00:54 rhel-9.2-aarch64-dvd.iso

I launched the container and executed pxeboot command like this:
# podman run -v /iso:/iso --pull always --replace --pid host --network host --user 0 --name bf -dit --privileged -v /dev:/dev quay.io/bnemeth/bf
# podman exec -it bf /pxeboot -e /iso/efiboot.img /iso/rhel-9.2-aarch64-dvd.iso

After the kernel and initrd were loaded (via TFTP), BF-2 tried to fetch files from /www/mnt,
but the HTTP process returned 404 error repeatedly. I solved this issue with following commands:
# podman exec -it bf sh
# ln -s /var/ftp/mnt /www/mnt


> Do you have insight on why it was working in the past?

I have no idea. If you have mounted the appropriate directory onto /www/mnt when launching the container, then it might work without th problem.